OSVROpenGL/app/src/main/jni/main.cpp

```diff
@@ -652,6 +626,12 @@ void stop() {
         delete gClientContext;
         gClientContext = nullptr;
     }
+
+    if(gServer) {
+        LOGI("[OSVR] Shutting down server...");
+        gServer->stop();
```

Comment:
Ah, ok, cool, so I did make the semantics more logical so that stop actually means stop. Forgive my mistake, I must have been thinking of a [different VR framework](http://www.vrjuggler.org/) that has a `waitForStop()` :+1: